src/Command/InfectionCommand.php

```diff
@@ -397,10 +425,15 @@ private function getCodeCoverageData(string $testFrameworkKey): LineCodeCoverage
 
     private function getTestFrameworkExtraOptions(string $testFrameworkKey): TestFrameworkExtraOptions
```

Comment:
I'm really not found of that one tbh. IMO that option should be parsed once at the beginning and re-injected elsewhere instead of relying on static state